components/style/properties/build.py

```diff
@@ -23,10 +23,12 @@ def main():
         abort(usage)
     product = sys.argv[1]
     output = sys.argv[2]
+    testing = len(sys.argv) > 3 and sys.argv[3] == "testing"
```

Comment:
We're not always passing this argument; style_tests itself contains a test that calls this without the third argument. I could add it, but it seems better to ignore.